src/utils/whitelist-tokens.ts

```diff
@@ -1,4 +1,15 @@
-export const whitelistedTokens = {
+export type AllowlistedToken = {
```

Comment:
Thanks so much! Only thing left is the filename...

Reply:
missed that!